### PR TITLE
Shavaiz/gp 340

### PR DESCRIPTION
**Pull request by shavaiz-hussain**
https://edlyio.atlassian.net/browse/GP-394

https://edlyio.atlassian.net/browse/GP-304
https://edlyio.atlassian.net/browse/GP-309

Added the badges envs (which are deleted during the conflicts resolution)

### CODE REVIEW

File: openedx/features/genplus_features/genplus_learning/signals.py [hunk 1]
```diff
@@ -58,18 +58,19 @@ def gen_class_changed(sender, instance, *args, **kwargs):
...
+#     pk_set = kwargs.pop('pk_set', None)
+#     if action == "post_add":
+#         if isinstance(instance, Class) and instance.program:
+#             genplus_learning_tasks.enroll_class_students_to_program.apply_async(
+#                 args=[instance.pk, instance.program.pk],
+#                 kwargs={
+#                     'class_student_ids': list(pk_set),
+#                 },
+#                 countdown=settings.PROGRAM_ENROLLMENT_COUNTDOWN
+#             )
```
Comment (zainvrk) [inline openedx/features/genplus_features/genplus_learning/signals.py:74]: comment?

**Review by zainvrk — DISMISSED** [review]
LGTM

File: openedx/features/genplus_features/genplus_learning/signals.py (hunk 1 above)
Comment (shavaiz-hussain) [reply openedx/features/genplus_features/genplus_learning/signals.py:74]: Yes @zainvrk i have added a TODO

Comment (zainvrk) [reply openedx/features/genplus_features/genplus_learning/signals.py:74]: ok

**Review by hasnain-saeed — APPROVED** [review]
@shavaiz-hussain LGTM 👍 please squash your commits, deploy this branch one more time and merge if everything is alright

**Pull request merged**